bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/discovery/HomematicDeviceDiscoveryService.java

```diff
@@ -74,11 +74,11 @@ protected void startScan() {
      */
     private void enableInstallMode() {
         try {
-            HomematicGateway gateway = thingHandler.getGateway();
             Thing bridge = thingHandler.getThing();
             ThingStatus bridgeStatus = bridge.getStatus();
 
-            if (ThingStatus.ONLINE == bridgeStatus) {
+            if (ThingStatus.ONLINE == bridgeStatus && thingHandler.getGateway() != null) {
+                HomematicGateway gateway = thingHandler.getGateway();
```

Comment:
Please revert this and only change :

if (ThingStatus.ONLINE == bridgeStatus && gateway != null) {

Reply:
fixed.